src/orders/components/OrderDraftDetailsProducts/TableLineForm.tsx

```diff
@@ -35,8 +30,15 @@ const TableLineForm: React.FC<TableLineFormProps> = ({
   const classes = useStyles({});
   const { id, quantity } = line;
 
+  const handleSubmit = (id: string, data: OrderLineInput) => {
+    if (!data || data.quantity < 1) {
+      return;
+    }
+    onOrderLineChange(id, { quantity: Math.floor(data.quantity) });
+  };
```

Comment:
I'm not sure about ux, shouldn't we display a warning, error, outline input with red color, disable save key or anything? 🤔

Reply:
In this case we have a debounce, and order line quantity is updated automatically after user leaves input.
It is ridiculous to put negative or fractional values as quantity and 0 makes little sense too because we have delete icon in the same line.

Mutation doesn't fire when `data.quantity < 1` but there is small default indicator here:
![image](https://user-images.githubusercontent.com/41952692/164676186-5f92a376-c952-49e2-b5e3-aedc2118e22b.png)